From: Egbert Eich <eich@suse.de>
Date: Fri Aug 22 10:07:22 2014 +0200
Subject: drm: Print EDID errors less obviously
Patch-mainline: to be upstreamed

References: bnc#892889
Signed-off-by: Egbert Eich <eich@suse.com>

Today DDC read code is quite reliable. The main source of EDID
read errors are bad KVM switches which don't seem to die out.
We do have quite reasonable fallbacks - read: default modes -
in place which will provide a reasonable graphics experience.
Don't clutter the text console with such EDID checksum errors.

Signed-off-by: Egbert Eich <eich@suse.de>
---
 drivers/gpu/drm/drm_edid.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/drm_edid.c b/drivers/gpu/drm/drm_edid.c
index 1cb5026..4f17842 100644
--- a/drivers/gpu/drm/drm_edid.c
+++ b/drivers/gpu/drm/drm_edid.c
@@ -1027,7 +1027,7 @@ bool drm_edid_block_valid(u8 *raw_edid, int block, bool print_bad_edid)
 		csum += raw_edid[i];
 	if (csum) {
 		if (print_bad_edid) {
-			DRM_ERROR("EDID checksum is invalid, remainder is %d\n", csum);
+			DRM_INFO("EDID checksum is invalid, remainder is %d\n", csum);
 		}
 
 		/* allow CEA to slide through, switches mangle this */
@@ -1055,8 +1055,8 @@ bool drm_edid_block_valid(u8 *raw_edid, int block, bool print_bad_edid)
 
 bad:
 	if (print_bad_edid) {
-		printk(KERN_ERR "Raw EDID:\n");
-		print_hex_dump(KERN_ERR, " \t", DUMP_PREFIX_NONE, 16, 1,
+		printk(KERN_INFO "Raw EDID:\n");
+		print_hex_dump(KERN_INFO, " \t", DUMP_PREFIX_NONE, 16, 1,
 			       raw_edid, EDID_LENGTH, false);
 	}
 	return false;
